igor-web/igor-web.gradle

```diff
@@ -36,7 +36,8 @@ dependencies {
     implementation "com.sun.xml.bind:jaxb-core:2.3.0.1"
     implementation "com.sun.xml.bind:jaxb-impl:2.3.2"
 
-    implementation "com.vdurmont:semver4j:3.1.0"
+    implementation "commons-io:commons-io"
+    implementation 'com.vdurmont:semver4j:3.1.0'
```

Comment:
Seen this dependency introduced in multiple places. It should be added to kork/spinnaker-dependencies.

Reply:
Hmm, this change doesn't actually use the `semver4j` library - this change is importing `commons-io`.  However, I think standardizing on a semver library is a good thing.

Created: https://github.com/spinnaker/kork/pull/784

The related `echo` PR did use that same `semver4j` library, and I can refactor that as well once there's a `kork` build for the above.